src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+  const hasComments = Object.keys(comments).length !== 0;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
```

Comment:
Let's imagine that the user will be `null`. Add a conditional rendering here 
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/99403198/319bdf22-3e28-49b7-b826-a07cb74b5e0b)